recipe/conda_build_config.yaml

```diff
@@ -41,8 +41,12 @@ m2w64_cxx_compiler:            # [win]
 m2w64_fortran_compiler:        # [win]
   - m2w64-toolchain            # [win]
 
-cuda_compiler:                 # [linux or win]
-  - nvcc                       # [linux or win]
+cuda_compiler:                 # [linux64 or win]
+  - nvcc                       # [linux64 or win]
```

Comment:
```suggestion
cuda_compiler:                 # [linux or win]
  - nvcc                       # [linux or win]
```

Reply:
This was changed in response to this suggestion above ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4407#discussion_r1182972314 )

Could you please share a bit more about why we want to revert that?

Should add the CUDA arch migrator now contains this key with appropriate length

Reply:
You are supposed to only change that for Line 46 and below.

Reply:
The selectors used should match that of `cuda_compiler_version`

Reply:
Agreed. Had another question about that below

Reply:
Sounds like we were ok with the suggestion below. So committed that

That said, if I'm still missing something here, please let me know